rsconnect/bundle.py

```diff
@@ -580,11 +580,11 @@ def make_quarto_source_bundle(
 
     base_dir = file_or_directory
     if not isdir(file_or_directory):
-        base_dir = basename(file_or_directory)
+        base_dir = dirname(file_or_directory)
```

Comment:
Without this, the bundle would be created with files that have incorrect paths in the case where the `deploy` command is called with a path or file path argument:
```
rsconnect deploy quarto <path/to/quarto.qmd>
```
The resulting bundle would try to add files `<path/to/quarto.qmd>` instead of `quarto.qmd`.